test/OpenTelemetry.AutoInstrumentation.Tests/Configurations/FileBased/Files/TestInstrumentationConfigurationFile.yaml

```diff
@@ -0,0 +1,22 @@
+instrumentation/development:
+  dotnet:
+    traces:
+      entityframeworkcore:
+        set_db_statement_for_text: true
+      graphql:
+        set_document: true
+      oraclemda: 
+        set_db_statement_for_text: true
+      aspnet:
+        capture_request_headers: "X-Key"
```

Comment:
Can we make this accept a list of string values rather than one csv string value?